doc/src/bond_over_grpc.md

```diff
@@ -13,13 +13,13 @@ supercedes the (now deprecated) Bond Comm framework.
 The Bond IDL has been extended to support the definition of
 [services](compiler.html#service-definition) and
 [generic services](compiler.html#generic-service). These definitions are
-used by the Bond compiler to generate:
+used by the Bond compiler to generate classes that provide:
 
-* service stubs that can be used as the base for implementing services'
+* service base that can be used as the basis for implementing services'
```

Comment:
Copy edit: either "services base<em>s</em> that can be used as the basis..." or "_a_ service base..."